include/tvm/tir/expr.h

```diff
@@ -888,66 +888,43 @@ class CallNode : public PrimExprNode {
     /*! \brief Intrinsic functions that are pure. */
     PureIntrinsic = 5
   };
-  /*! \brief The name of the function/intrinsic. */
-  String name;
+  /*!
+   * \brief The operator(function) being invoked
+   *
+   *  - It can be tvm::Op which corresponds to the primitive operators(intrinsics).
+   *  - It can also be another function in the IRModule (GlobalVar).
+   */
+  RelayExpr op;
```

Comment:
Do you think we should someday move GlobalVar out of RelayExpr?

Reply:
Any suggested alternatives and rationale?